datatypes/src/collections/feature_collection_builder.rs

```diff
@@ -299,14 +326,20 @@ where
                     std::mem::size_of::<u8>()
                 } else if builder.as_any().is::<StringBuilder>() {
                     0 // TODO: how to get this dynamic value
+                } else if builder.as_any().is::<BooleanBuilder>() {
+                    // arrow buffer internally packs 8 bools in 1 byte
+                    bool_bytes = arrow::util::bit_util::ceil(builder.len(), 8);
+                    0
```

Comment:
I do not return the value directly because it would get multiplied by len() in `let values_size = builder.len() * data_type_size;` so the resulting size would be too large

Reply:
343f15f
I reworked the if's to make clearer whats happening